programs/bpf/c/src/ser/ser.c

```diff
@@ -0,0 +1,34 @@
+/**
+ * @brief Example C-based BPF sanity rogram that prints out the parameters
+ * passed to it
+ */
+#include <solana_sdk.h>
+
+extern uint64_t entrypoint(const uint8_t *input) {
+  SolAccountInfo ka[1];
+  SolParameters params = (SolParameters){.ka = ka};
+
+  sol_log(__FILE__);
+
+  if (!sol_deserialize(input, &params, SOL_ARRAY_SIZE(ka))) {
+    return ERROR_INVALID_ARGUMENT;
+  }
+
+  char ka_data[] = {1, 2, 3};
+  SolPubkey ka_owner;
+  sol_assert(params.ka_num == 2);
+  for (int i = 0; i < 2; i++) {
+    sol_assert(*params.ka[i].lamports == 42);
+    sol_assert(!sol_memcmp(params.ka[i].data, ka_data, 4));
+    sol_assert(SolPubkey_same(params.ka[i].owner, &ka_owner));
```

Comment:
This test seems to fail consistently at this line.  Why should it work if local struct `ka_owner` is not initialized?  What am I missing?..